code/client/clrcore-v2/Interop/ReferenceFunctionManager.cs

```diff
@@ -25,6 +25,11 @@ public Function(MsgPackFunc method, byte[] id)
 
 		private static Dictionary<int, Function> s_references = new Dictionary<int, Function>();
 
+		static ReferenceFunctionManager()
+		{
+			MsgPackReferenceRegistrar.CreateFunc = Create;
```

Comment:
I had to manually add the new `MsgPackReferenceRegistrar.cs` file in CitizenFX.Core csproj, but VS didn't register that, i suspect that when the module is loaded after merge it'll be automatically added as part of the msgpack library itself